mlflow/utils/autologging/__init__.py

```diff
@@ -315,7 +312,7 @@ def _check_version_in_range(ver, min_ver, max_ver):
 
 
 def _load_version_file_as_dict():
-    version_file_path = resource_filename(__name__, "../ml-package-versions.yml")
+    version_file_path = resource_filename(__name__, "../../ml-package-versions.yml")
```

Comment:
As a result of https://github.com/mlflow/mlflow/pull/4184/files#r596149139, `ml-package-versions.yml` is now an additional directory above this module.